docs/content/direct/architecture.md

```diff
@@ -50,11 +50,11 @@ that are put in a mailbox namespace are delivered to the matching WEC.
 on all WECs and sets the RBAC permissions for it using the OCM add-on framework.
 
 - *OCM Agent*: This module registers the WEC to the OCM Hub, watches for 
-ManifestWorks and unwraps and syncs the objects into the WEC.
+[ManifestWork.v1.work.open-cluster-management.io](https://github.com/open-cluster-management-io/api/blob/v0.11.0/work/v1/types.go#L17) objects and unwraps and syncs the objects into the WEC.
```

Comment:
I see that we are using v0.12.0 for the OCM APIs. Do we want to make the version consistent if we put a link to the OCM source, or it doesn't matter? (There seems to be no diff though between 0.11.0 and 0.12.0).

Reply:
Thanks for caching that. I have pushed a rev that corrects the versions.